### ISSUE TITLE
svgo breaks image

### ISSUE THREAD

**Issue by ikokostya** [svg/svgo#329]
```
[kostya@arch svgo-test]$ svgo -v
util.error: Use console.error instead
0.5.0
[kostya@arch svgo-test]$ svgo orig.svg -o min.svg
util.puts: Use console.log instead

util.puts: Use console.log instead
Done in 76 ms!
util.puts: Use console.log instead
2.116 KiB - 70.2% = 0.631 KiB
```

Original image:

![orig](https://cloud.githubusercontent.com/assets/328125/6770325/d9b23c8a-d0cd-11e4-8de5-47089b773880.png)

```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg width="40px" height="40px" viewBox="0 0 40 40" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
    <!-- Generator: Sketch 3.0.4 (8054) - http://www.bohemiancoding.com/sketch -->
    <title>Гостиницы</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Схема-поиска" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
        <g id="Карточка-Города-(Турция)" sketch:type="MSLayerGroup" transform="translate(-260.000000, -302.000000)">
            <g id="Поиск-рядом" transform="translate(16.000000, 269.000000)" sketch:type="MSShapeGroup">
                <g id="Рестораны,-Кафе.." transform="translate(3.000000, 33.000000)">
                    <g id="Гостиницы" transform="translate(241.000000, 0.000000)">
                        <circle id="Oval-10" fill="#F6F5F3" cx="20" cy="20" r="20"></circle>
                        <g id="Group-copy-4" transform="translate(11.000000, 12.500000)" fill="#7C6958">
                            <path d="M4,8.5 L8,8.5 C8,8.5 7.99999985,8.5 8,7 C8,5.5 4,5.5 4,7 C3.99999987,8.5 4,8.5 4,8.5 Z" id="Path-264-copy"></path>
                             <path d="M16,12.5 C17.3080764,12.5 18,11.8284271 18,11.5 C18,10.1715729 17.3080764,9.5 16,9.5 L3,9.5 C1.69192357,9.5 1,10.1715729 1,11.5 C1,11.8284271 1.69192357,12.5 3,12.5 L16,12.5 Z" id="Line-copy"></path>
                             <path d="M15,13.5 C15,14.3284271 15.6715729,15 16.5,15 C17.3284271,15 18,14.3284271 18,13.5 L18,4.5 C18,3.67157288 17.3284271,3 16.5,3 C15.6715729,3 15,3.67157288 15,4.5 L15,13.5 Z" id="Line-copy"></path>
                             <path d="M0,13.5 C0,14.3284271 0.671572875,15 1.5,15 C2.32842712,15 3,14.3284271 3,13.5 L3,1.5 C3,0.671572875 2.32842712,0 1.5,0 C0.671572875,0 0,0.671572875 0,1.5 L0,13.5 Z" id="Line-copy"></path>
                        </g>
                    </g>
                </g>
            </g>
        </g>
    </g>
</svg>
```

After:

![min](https://cloud.githubusercontent.com/assets/328125/6770327/e05ef47e-d0cd-11e4-9e8b-4d45473e326d.png)

```svg
<svg width="40" height="40" viewBox="0 0 40 40" xmlns="http://www.w3.org/2000/svg"><title>Гостиницы</title><desc>Created with Sketch.</desc><g fill="none" fill-rule="evenodd"><g><g><g><g transform="translate(0 0)"><circle fill="#F6F5F3" cx="20" cy="20" r="20"/><g fill="#7C6958"><path d="M15 21h4v-1.5c0-1.5-4-1.5-4 0V21z"/><path d="M27 25c1.308 0 2-.672 2-1 0-1.328-.692-2-2-2H14c-1.308 0-2 .672-2 2 0 .328.692 1 2 1h13zM26 26c0 .828.672 1.5 1.5 1.5S29 26.828 29 26v-9c0-.828-.672-1.5-1.5-1.5S26 16.172 26 17v9zM11 26c0 .828.672 1.5 1.5 1.5S14 26.828 14 26V14c0-.828-.672-1.5-1.5-1.5S11 13.172 11 14v12z"/></g></g></g></g></g></g></svg>
```

**Comment by ikokostya** [comment]
Disabling `mergePaths` plugin resolves this issue.

**Comment by GreLI** [comment]
Fixed in e1060df.

**Comment by GreLI** [comment]
[v0.5.1](https://github.com/svg/svgo/releases/tag/v0.5.1) with this fix has been just published.
